cdc/processor/sinkmanager/table_sink_worker.go

```diff
@@ -312,11 +312,6 @@ func (w *sinkWorker) handleTask(ctx context.Context, task *sinkTask) (finalErr e
 			pendingTxnSize = 0
 			batchID = 1
 		}
-		if pos.IsCommitFence() {
```

Comment:
Why do we remove it?

Reply:
In `doEmit` the condition about `CommitFence` is handled correctly. So here it's unnecessary to check `CommitFence` again.